### PR TITLE
Change ganache to geth

### PR DESCRIPTION
**Pull request by dongrie**
Changed the Ethereum chain from ganache to geth

### CODE REVIEW

File: tests/chains/ethereum/geth/run.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+/usr/local/bin/geth --password /root/geth.password \
+  --unlock "0" --syncmode full --gcmode archive \
+  --authrpc.vhosts "*" --authrpc.addr "0.0.0.0" --http --http.addr "0.0.0.0" --http.api web3,eth,net,personal,miner,txpool,debug --http.corsdomain '*' \
+  --ws --ws.api eth,net,web3,personal,txpool --ws.addr "0.0.0.0" --ws.port "8546" --ws.origins '*' \
+  --datadir /root/.ethereum --networkid "9999" --nodiscover \
```
Comment (siburu) [inline tests/chains/ethereum/geth/run.sh:7]: why 9999?

**Review by siburu — COMMENTED** [review]
@dongrie I added some comments

File: tests/chains/ethereum/geth/run.sh (hunk 1 above)
Comment (dongrie) [reply tests/chains/ethereum/geth/run.sh]: Removed networkid
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/89df50ec9a581fabbbb4e7054b4e1911487b6eee

File: tests/chains/ethereum/geth/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM ethereum/client-go:v1.11.5
```
[the remark this answers is not in the document]
Comment (dongrie) [reply tests/chains/ethereum/geth/Dockerfile]: Use latest version v1.13.14
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/89df50ec9a581fabbbb4e7054b4e1911487b6eee

File: .github/workflows/v0.4-build.yml [hunk 3]
```diff
@@ -79,4 +79,4 @@ jobs:
         if: steps.cache-docker-ethereum.outputs.cache-hit != 'true'
         working-directory: ./tests/scripts
         run: |
-          ./save_docker_images $CACHE_DOCKER_ETHEREUM_DIR ethereum-ganache0:latest ethereum-ganache1:latest
+          ./save_docker_images $CACHE_DOCKER_ETHEREUM_DIR ethereum-ibc0:latest ethereum-ibc1:latest
```
[the remark this answers is not in the document]
Comment (dongrie) [reply .github/workflows/v0.4-build.yml]: change to geth{0,1}
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/89df50ec9a581fabbbb4e7054b4e1911487b6eee

File: tests/chains/ethereum/contracts/contracts/.gitkeep
Comment (siburu) [inline tests/chains/ethereum/contracts/contracts/.gitkeep:1]: Isn't this file necessary?

File: tests/chains/ethereum/contracts/package.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "contracts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "TBD",
```
Comment (siburu) [inline tests/chains/ethereum/contracts/package.json:9]: "main" and "scripts" are unnecessary.
"license" should be "Apache-2.0".

File: tests/chains/ethereum/contracts/scripts/deploy.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+if (require.main === module) {
+  main()
+    .then(() => process.exit(0))
+    .catch((error) => {
+      console.error(error);
+      process.exit(1);
+    });
+}
+
+exports.deployIBC = deployIBC;
```
Comment (siburu) [inline tests/chains/ethereum/contracts/scripts/deploy.js:73]: Is it necessary to export `deployIBC`?

**Review by siburu — COMMENTED** [review]
@dongrie I put some comments and questions.

File: tests/chains/ethereum/contracts/contracts/.gitkeep
Comment (dongrie) [reply tests/chains/ethereum/contracts/contracts/.gitkeep:1]: Remove unused git file
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/b7011d80c90f3c54f4f7f76106637736b312d228

File: tests/chains/ethereum/contracts/package.json (hunk 4 above)
Comment (dongrie) [reply tests/chains/ethereum/contracts/package.json]: Add license
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/b7011d80c90f3c54f4f7f76106637736b312d228

File: tests/chains/ethereum/contracts/scripts/deploy.js (hunk 5 above)
Comment (dongrie) [reply tests/chains/ethereum/contracts/scripts/deploy.js]: Remove unnecessary exports
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/b7011d80c90f3c54f4f7f76106637736b312d228

**Review by siburu — APPROVED** [review]
@dongrie LGTM, thank you!

**Pull request merged**